integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkCommonIT.java

```diff
@@ -56,32 +51,14 @@ private static String getInsertWithoutPartitionSql(String tableName, String valu
                   DataTypes.createStructField("col2", DataTypes.StringType, true))),
           "struct(1, 'a')");
 
-  // Use a custom database not the original default database because SparkIT couldn't read&write
-  // data to tables in default database. The main reason is default database location is
-  // determined by `hive.metastore.warehouse.dir` in hive-site.xml which is local HDFS address
-  // not real HDFS address. The location of tables created under default database is like
-  // hdfs://localhost:9000/xxx which couldn't read write data from SparkIT. Will use default
-  // database after spark connector support Alter database xx set location command.
-  @BeforeAll
-  void initDefaultDatabase() {
```

Comment:
same reason for SparkEnvIT, I prefer not override `initDefaultDatabase` and `init`.  Is there any scene what the catalog should use a different default databases?

Reply:
got it. Using the same default database is ok.

Reply:
fixed.